plugins/Jobs.cpp

```diff
@@ -1026,7 +1109,11 @@ string BedrockPlugin_Jobs::_constructNextRunDATETIME(const string& lastScheduled
     }
 
     // Validate the sqlite date modifiers
-    // See: https://www.sqlite.org/lang_datefunc.html
+    if (!_isValidSQLiteDateModifier(SComposeList(parts))){
+        SWARN("Syntax error, failed parsing repeat");
+        return "";
+    }
+
     for (const string& part : parts) {
```

Comment:
If you created `_isValidSQLiteDateModifier`, this seems unneeded and doing the same thing.